includes/templates/reader-activation-emails/otp.php

```diff
@@ -9,49 +9,54 @@
 	<!-- /wp:button --></div>
 	<!-- /wp:buttons -->
-
 	<!-- wp:paragraph -->
 	<p>' . __( 'Or copy and paste the link in your browser:', 'newspack' ) . ' <a href="*MAGIC_LINK_URL*" data-type="URL" data-id="*MAGIC_LINK_URL*">*MAGIC_LINK_URL*</a></p>
 	<!-- /wp:paragraph -->
-
 	<!-- wp:paragraph -->
 	<p>' . __( 'If you did not request this code, please ignore this email.', 'newspack' ) . '</p>
 	<!-- /wp:paragraph --></div>
-	<!-- /wp:group -->
+	<!-- /wp:group -->' .
 
-	<!-- wp:group {"style":{"spacing":{"padding":{"top":"56px","bottom":"56px","left":"56px","right":"56px"}},"color":{"background":"' . esc_attr( $primary_color ) . '"}},"className":"has-primary-background-color","layout":{"type":"constrained"}} -->
-	<div class="wp-block-group has-primary-background-color has-background" style="background-color:' . esc_attr( $primary_color ) . ';padding-top:56px;padding-right:56px;padding-bottom:56px;padding-left:56px"><!-- wp:social-links {"iconBackgroundColor":"' . esc_attr( $primary_color ) . '","iconBackgroundColorValue":"' . esc_attr( $primary_color ) . '","className":"is-style-filled-white","layout":{"type":"flex","flexWrap":"nowrap"}} -->
-	<ul class="wp-block-social-links has-icon-background-color is-style-filled-white">' . $social_links . '</ul>
+	// Footer.
+	'<!-- wp:group {"style":{"spacing":{"padding":{"top":"56px","bottom":"56px","left":"56px","right":"56px"}},"color":{"background":"' . esc_attr( $primary_color ) . '"},"elements":{"link":{"color":{"text":"' . esc_attr( $primary_text_color ) . '"}}}},"textColor":"' . esc_attr( $primary_text_color ) . '","className":"has-primary-background-color","layout":{"type":"constrained"}} -->
+	<div class="wp-block-group has-primary-background-color has-' . esc_attr( $primary_text_color ) . '-color has-text-color has-background has-link-color" style="background-color:' . esc_attr( $primary_color ) . ';padding-top:56px;padding-right:56px;padding-bottom:56px;padding-left:56px">
+	<!-- wp:social-links {"iconColor":"' . esc_attr( $primary_text_color ) . '","iconColorValue":"' . ( $primary_text_color === 'black' ? '#000000' : '#ffffff' ) . '","iconBackgroundColor":"' . esc_attr( $primary_color ) . '","iconBackgroundColorValue":"' . esc_attr( $primary_color ) . '","className":"is-style-filled-white","layout":{"type":"flex","flexWrap":"nowrap"}} -->
+	<ul class="wp-block-social-links has-icon-color has-icon-background-color is-style-filled-white">' . $social_links . '</ul>
 	<!-- /wp:social-links -->
-
 	<!-- wp:paragraph {"fontSize":"small"} -->
 	<p class="has-small-font-size">' . sprintf( /* Translators: 1: site title 2: site base address. */ __( '%1$s - %2$s', 'newspack' ), '<strong>*SITE_TITLE*</strong>', '*SITE_ADDRESS*' ) . '<br>' . sprintf( /* Translators: 1: link to site url. */ __( 'You received this email because you requested to sign in to %s', 'newspack' ), '<a href="*SITE_URL*">*SITE_URL*</a>' ) . '</p>
```

Comment:
I've just noticed – the text domain in the translation functions is wrong. Should be `'newspack-plugin'`.

Reply:
Good catch! Fixed in https://github.com/Automattic/newspack-plugin/pull/3033/commits/6510cc350130095d99547fc49098875bdd880ede